examples/motor-imagery/helpers/frequencybandselection_helpers.py

```diff
@@ -0,0 +1,226 @@
+"""
+=================================
+ Frequency Band Selection Helpers
+=================================
+
+This file contains helper functions for the frequency band selection example
+"""
+
+import numpy as np
+from mne import Epochs, events_from_annotations
+
+from pyriemann.estimation import Covariances
+from pyriemann.classification import class_distinctiveness
+
+
+def freq_selection_class_dis(raw, freq_band=[5., 35.], sub_band_width=4,
```

Comment:
```suggestion
def freq_selection_class_dis(raw, freq_band=(5., 35.), sub_band_width=4,
```
you should avoid mutable parameters in function signatures

Reply:
I see. I will keep that in mind.